include/srtp.h

```diff
@@ -1484,6 +1697,18 @@ srtp_err_status_t srtp_set_debug_module(char *mod_name, int v);
  */
 srtp_err_status_t srtp_list_debug_modules(void);
 
+/**
+ * @brief srtp_get_protect_data_added_length(policy, use_mki, mki_index, is_rtcp, length)
+ *
+ * Determines the length of the amount of data Lib SRTP will add to the 
+ * packet during the protect process. The length is returned in the length parameter
+ *
+ * returns err_status_ok on success, err_status_bad_mki if the MKI index is invalid
+ *
+ */
+srtp_err_status_t srtp_get_protect_data_added_length(const srtp_policy_t *policy, uint32_t use_mki,
```

Comment:
I can see why having a more dynamic way of determining the required size of the buffer could be useful especially with a possible mki size of 128 bytes but is it really needed? Why not just change the definition of SRTP_MAX_TRAILER_LEN to be SRTP_MAX_TAG_LEN + SRTP_MAX_MKI_LEN?  If you look at the current documentation for SRTP_MAX_TRAILER_LEN, it already states that it will include the auth tag and the mki size. 

If it is decided that this functions is far more useful then perhaps remove the SRTP_MAX_TRAILER_LEN define altogether and instead rename this to srtp_get_protect_trailer_length, in which case I would also remove the is_rtcp argument and have a second function called srtp_get_protect_rtcp_trailer_length() keeping it in line with the rest of the api.  Also why operate on a policy rather than a session?